### PR TITLE
upgrade nginx version for examples

### PR DESCRIPTION
**Pull request by inductor**
nginx 1.9 is a very old version so decided to upgrade it as an example

### CODE REVIEW

File: content/ja/examples/service/access/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM nginx:1.9.14
+FROM nginx:1.17.3
```
Comment (xiangpengzhao) [inline content/ja/examples/service/access/Dockerfile]: @sftim IIRC, we just change the en content and let localization team change the related localized content, right?

**Comment by zacharysarah** [comment]
@inductor :wave: Could you please open separate PRs for changes in English and Japanese? Thanks in advance!

/hold

**Pull request closed without merging**